datastore/google/cloud/datastore/connection.py

```diff
@@ -220,6 +237,44 @@ def allocate_ids(self, project, request_pb):
                          _datastore_pb2.AllocateIdsResponse)
 
 
+def _grpc_catch_rendezvous(to_call, *args, **kwargs):
```

Comment:
I don't understand the comment. The control would instead be handed over to the context manager.